+++
abstract = "Motivation: DNA methylation is an epigenetic change occurring in genomic CpG sequences that contribute to the regulation of gene transcription both in normal and malignant cells. Next-generation sequencing has been used to characterize DNA methylation status at the genome scale, but suffers from high sequencing cost in the case of whole-genome bisulfite sequencing, or from reduced resolution (inability to precisely define which of the CpGs are methylated) with capture-based techniques. Results: Here we present a computational method that computes nucleotide-resolution methylation values from capture-based data by incorporating fragment length profiles into a model of methylation analysis. We demonstrate that it compares favorably with nucleotideresolution bisulfite sequencing and has better predictive power with respect to a reference than window-based methods, often used for enrichment data. The described method was used to produce the methylation data used in tandem with gene expression to produce a novel and clinically significant gene signature in acute myeloid leukemia. In addition, we introduce a complementary statistical method that uses this nucleotide-resolution methylation data for detection of differentially methylated features. Availability: Software in the form of Python and R scripts is available at http://bioserv.mps.ohio-state.edu/premer and is free for non-commercial use"
abstract_short = "Short version of abstract."
date = "2014-08-12T16:37:31+02:00"
image = ""
image_preview = ""
math = false
publication = "PrEMeR-CG: inferring nucleotide level DNA methylation values from MethylCap-seq data"
publication_short = "PrEMeR-CG: inferring nucleotide level DNA methylation values from MethylCap-seq data"
selected = false
title = "PrEMeR-CG: inferring nucleotide level DNA methylation values from MethylCap-seq data"
url_code = ""
url_dataset = ""
url_pdf = "https://watermark.silverchair.com/btu583.pdf?token=AQECAHi208BE49Ooan9kkhW_Ercy7Dm3ZL_9Cf3qfKAc485ysgAAAuIwggLeBgkqhkiG9w0BBwagggLPMIICywIBADCCAsQGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQMZE4idQ2pWdmTgEA9AgEQgIIClV68rSyh9Puas3zAL_4TZeg8oxA6_xiJc6JrkIr-0apPw_NvGV9KBXUc4IVmbRBotnQ1lMVMCv0SHdH-0tR5Aq52yZiosDXfH33m90HLvOWUBJyD-1kGKHvR3TpZDHZc5mQ8LolimIWA1uzAuEoMSHUQE-2RchhSRc7hjZobOTvGoM4HpQ_SoC0H4t5cEny8U5Vva4WGFUAtyopynqRBbQeXSRRo_OSDNUycBLZlux2lzgKXZ4V9fSOrHLVfLW8k2PcCcO-Aq-zJqoGtP-0zopGWs-Nu3NX3BlE0ve2Hn2Zm6mmdkR8xJD33gUbUMwUAms3TACO4x2Dj8uQBaC-QS9Ui7qRNbE5ebSu13SqQe8rJogVufiUyATmVR973837EeL4TD0qLr0y2uIzfbGlXXW6QDpdZvJHFQijEDv5DfnyUIV8TiXmtmO81OsHeQ4PLzKJmIljTJFnBbhtx-3qqJXD5Uyhhu3aDGRQZm61elPCmLigfJZI--4rrWHTeQgve614bpuc89UtTZL4iY28zHTVaeKmPu6T0A5NKpxyAxMzZKwPTK5xusctMz7Tbp69OmxIE0fdIXy78DkZB8TTUyvg4oMTAxg0Ul_1TqojIKhPvV1maXKgVpiKqaFZ_OMhP6LtcbK86Y0y_kVPkRwXQnG3yG1kTwUxCRcHSgBSFLmLREckRkBLQuu0Z-qyIVtpwOtHPEbv5Am2ts_Oqc29i-0Sgv5wmIy3z4v_QRGnXS1FVkF4N8WbgO_Z2AYMmjsSny25_JTL8O9DhLx4UJKHyi5ruFgD5ZsLEjMiKfMzcPZb3emboQeCyoenSSH-cYk1S7uuC0mcUzziZhOQfoD9CuAXE_cwJ0ANdPbuzCwp9iVyEuNu7Mrg"
url_project = "/project/research"
url_slides = ""
url_video = ""

[[authors]]
    name = "David E. Frankhouser"
    is_member = false
[[authors]]
    name = "James S. Blachly"
    is_member = true
[[authors]]
    name = "Jincheol Park"
    is_member = false
[[authors]]
    name = "Mike W. Zoller"
    is_member = false
[[authors]]
    name = "Javkhlan-Ochir Ganbat"
    is_member = false
[[authors]]
    name = "John Curfman"
    is_member = false
[[authors]]
    name = "John C. Byrd"
    is_member = false
[[authors]]
    name = "Shili Lin"
    is_member = false
[[authors]]
    name = "Guido Marcucci"
    is_member = false
[[authors]]
    name = "Pearlly Yan"
    is_member = false
[[authors]]
    name = "Ralf Bundschuh"
    is_member = false
+++